packages/mantle/src/components/tabs/tabs.tsx

```diff
@@ -54,23 +56,36 @@ const TabsStateContext = createContext<TabsStateContextValue>({
  */
 const Root = forwardRef<
 	ComponentRef<typeof TabsPrimitiveRoot>,
-	ComponentPropsWithoutRef<typeof TabsPrimitiveRoot>
->(({ className, children, orientation = "horizontal", ...props }, ref) => (
-	<TabsPrimitiveRoot
-		className={cx(
-			"flex gap-4",
-			orientation === "horizontal" ? "flex-col" : "flex-row",
+	ComponentPropsWithoutRef<typeof TabsPrimitiveRoot> & {
+		appearance?: "classic" | "pill";
```

Comment:
we will want to document this prop w/ jsdoc

```suggestion
    /**
     * Something something gippity please fill me out
     * @default "classic"
     */
		appearance?: "classic" | "pill";
```